src/catalog/src/durable/impls/migrate.rs

```diff
@@ -61,6 +63,14 @@ impl OpenableDurableCatalogState for CatalogMigrator {
         bootstrap_args: &BootstrapArgs,
         deploy_generation: Option<u64>,
     ) -> Result<Box<dyn DurableCatalogState>, CatalogError> {
+        // Handle emergency stash before opening persist.
+        if self.direction == Direction::EmergencyStash {
+            return self
+                .openable_stash
+                .open_savepoint(boot_ts.clone(), bootstrap_args, deploy_generation.clone())
```

Comment:
I think the calls to `clone` are superfluous

Reply:
Removed